### DISCUSSION TITLE
Encoder-decoder inference with fast kernels

### DISCUSSION THREAD

**Discussion by saikrishb** [deepspeedai/DeepSpeed#2469] [General]
Hi, I'm interested in using deepspeed to run inference on a [BART model](https://huggingface.co/docs/transformers/model_doc/bart), with kernel injection. I see this PR open for some time now. Any ETA on when this feature would be available?  
https://github.com/microsoft/DeepSpeed/pull/2451

@RezaYazdaniAminabadi

**Comment by RezaYazdaniAminabadi** [comment]
Hi @saikrishb 

Thanks for your interest. I am actively working on this. It should be out most likely by beginning or middle of december.
Best,
Reza

**Reply by drxmy** [reply]
> Hi @saikrishb
> 
> Thanks for your interest. I am actively working on this. It should be out most likely by beginning or middle of december. Best, Reza

I am new to inference optimization but trying to deploy a mt5-xxl.  Is there something else i could do after using this awesome kernel injection? 
Really appreciate it if you could give any advice.
